src/pygeosx/pygeosx.cpp

```diff
@@ -376,12 +393,22 @@ PyInit_pygeosx()
     return nullptr;
   }
 
+  if( !LvArray::python::addTypeToModule( module, geosx::python::getPySolver2Type(), "Solver2" ) )
```

Comment:
What's `Solver2`? I see it here but nowhere else.

Reply:
Solver2 is another way to create PySolver structure in the way you created PyWrapper and PyGroup

Reply:
So then all references to it should be removed right?

Reply:
Done in new commit. Sorry for that.